### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by NataJenkins**
TNW website clone using HTML and CSS.
media queries used for responsiveness.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+-HTML
+-CSS
+
+## Live Demo
+
+[Live Demo Link](https://410AngelaVu.github.io/Responsive-Design/)
+
+## Authors
+
+👤 **Author1**
```
Comment (elmejdki) [inline README.md]: use your full names here

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+- Github: [@NataJenkins](https://github.com/Takhmina175)
+- Twitter: [@NataJenkins](https://twitter.com/NataJenkins)
+- Linkedin: [linkedin Profile](https://www.linkedin.com/in/natalia-macias-a11a20187/)
+
+👤 **Author2**
```
Comment (elmejdki) [inline README.md]: use your full names here

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1077 @@
...
+  margin-left: 20px;
+}
+
+.white-navbar ul li {
+  color: hsla(0, 0%, 100%, 0.6);
+  margin-right: 17px;
+}
+
+.countdown {
+  color: red !important;
```
Comment (elmejdki) [inline style.css]: remove all !important keywords they are not allowed

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1077 @@
...
+
+.white-nav-logo {
+  height: 10px;
+}
+
+.tickets-btn {
+  background-color: #0cc;
+  padding: 6px;
+  border-radius: 5px;
+  color: white !important;
```
Comment (elmejdki) [inline style.css]: remove all !important keywords they are not allowed

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job so far guys :clap: :100: :+1:, your clone looks in too many ways like the original, but you still need to work :hammer: :wrench: :toolbox: more on it, in order to get it approved :checkered_flag:

- [x] below the hero section your articles should take the same width:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] You are messing the latest funding section:

image

- [x] remove the borders from the latest deals articles, the borders are not necessary:
image

- [x] you are having a burger menu on desktop view please remove it, and show it only on mobile view as in the original:
image

- [x] we are still on desktop view and your layout changes please fix your media queries to match only tablet and desktop views pixels

|*layout*|
|:---:|
| image |
| image |

- [ ] CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))

* No small break point as it is mobile first and the base styles are for the mobile screen. ([Screenshot on mobile first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
* medium: min-width around 768px
* large: min-width 1024px

- [x] Fix the typography here on tablet view:
image

- [x] Fix the big image on the latest deals section on tablet view and also use some margin to separate your articles, also we should have three articles on a row, not four:
image

- [ ] Please fix the header it should match the original:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] This section should have two articles on a row, not 3, on tablet view of course:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] you need to put some work on the footer in mobile view:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] Fix all the warning and error in the HTML validator, you can fix this by adding a heading to the article tag and use display: none;
image

- [x] put your images on the assets/images folder instead of the root directory:
image

- [x] divide your HTML page into a header, main and footer section using these semantic tags.

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you are stuck or have any questions don't hesitate to contact me on slack **@Zakariae El Mejdki**_

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1975 @@
...
+            <div class="li-div-n">
+              <div class="deal d-flex">
+                <img alt="deal"
+                  src="https://cdnp3.stackassets.com/5520ced2e2087194c396763c173fc619db5a52a3/store/435968f3d598af6f168bca2a38af51e83929a031333582e96a49cea4fa6e/sale_29727_primary_image.jpg">
+                <div class="deal-info">
+                  <h4>
+                    <a href="#" class="black">BetterMe Home Workout & Diet: Lifetime Subscription</a>
+                  </h4>
+                  <div class="deal-price d-flex">
+                    <span class="old as-center">$1200</span>
```
Comment (expjazz) [inline index.html:1897]: - [ ] The layout in the tablet view needs to be 3 articles per columns.

| Yours | Original |
|:---:|:---:|
|image | image |

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1975 @@
...
+      <p>See more | Get updates by email | Add info</p>
+    </div>
+    <div class="latest-funding">
+      <div class="funding-1">
+        <div class="plus">
+          <img src="assets/Images/kocka.png" alt="" class="jaja">
+
+          <div class="f-1">
+
+            <p>EverCompliant</p>
```
Comment (expjazz) [inline index.html:167]: - [ ] This needs to be centralized (and occupy more space) on the tablet screen size.

| Yours | Original |
|:---:|:---:|
|image|image  |

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1975 @@
...
+  <main>
+    <article class="section-2">
+      <h4 class="l-h4 cp">Latest news &nbsp;<i class="fas fa-chevron-right"></i></h4>
+      <div class="grid-article-2">
+
+        <div class="div-g-1 cp">
+
+          <img src="assets/Images/kreten-1.jpg" alt="" class="slike">
+          <p class="main">Tesla dumped $5B worth of its stock in just 4 days — and traders hate it</p>
+          <p class="main-little">David Canellis4 hours ago</p>
```
Comment (expjazz) [inline index.html:116]: - [ ] The height in the tablet screen size should be a little smaller.

| Yours | Original |
|:---:|:---:|
|image| image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hey :+1: 

You did a great job here, and the project is almost finished. Generally, you still need to improve the layout of the tablet screen size. Kindly check the inline comments for more details.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello Team! I can see you worked really hard on this project. I reviewed your [project](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)  and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
Now you can merge the branches and move forward with your curriculum. 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 
**Happy Coding**:smiley:
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) @slack: arslanbisharat

**Pull request merged**